charmdir.go

```diff
@@ -454,19 +453,69 @@ func (m NopLogger) Infof(message string, args ...interface{}) {
 
 }
 
+type vcsCMD struct {
+	vcsType       string
+	args          []string
+	usesTypeCheck func() bool
+	errHandler    func(err error, vcsType string, path string)
+	charm         *CharmDir
```

Comment:
I don't think `charm *CharmDir` is required here.

Reply:
yeah you are right, had this initially in there and somehow lost it :D